crowbar_framework/lib/crowbar/upgrade_timeouts.rb

```diff
@@ -0,0 +1,37 @@
+require "yaml"
+
+module Crowbar
+  class UpgradeTimeouts
+    def values
+      @timeouts_config = begin
+        YAML.load_file("/etc/crowbar/upgrade_timeouts.yml")
```

Comment:
Basically I did it this way so the user does not need to restart crowbar if they add a new value to the config file.

Note that this is done this way because I assume that once rails is started it preloads all this files, so a "static" load_file would only be called once and the user would need to restart the whole server just for this.

If this is not the case, let me know so I can implement it a different way!